src/Apps/Consign/Routes/MarketingLanding/Components/Header.tsx

```diff
@@ -48,16 +50,48 @@ export const Header: React.FC = () => {
               auction, private sale, or direct listing on Artsy.
             </Text>
 
-            <Button
-              // @ts-ignore
-              as={RouterLink}
-              variant="primaryWhite"
-              to="/sell/submission/artwork-details"
-              onClick={trackSubmitClick}
-              mb={[4, 0]}
-            >
-              Submit an Artwork
-            </Button>
+            {!showGetInTouchCTA ? (
+              <Flex flexDirection={["column", "row"]} maxWidth="450px">
```

Comment:
Setting the width here explicitly to give the buttons the same width setting `flex: 1`.